packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -141,6 +151,8 @@ class TestCommand extends FlutterCommand {
     }
 
     commandValidator();
+    final List<String> names = argResults['name'];
```

Comment:
If the user didn't specify the argument, will this value be `null` or the empty list (since `allowMultiple` was true)?

If it's `null`, then the default values in `runTests()` won't work, and you'll get an NPE.